File: Day-08/Pattern5.js
function PatternFive(n) {
  let i = 1;
  let count = 1;
  while (i <= n) {
    let j = 1;
    let str = "";
    while (j <= n) {
      str += count;
      count++;
      j++;
    }
    i++;
    console.log(str);
  }
}

PatternFive(3);

File: Day-08/Pattern5.py
n = int(input())

def PatternFive(n):
    i= 1
    count = 1
    while(i <= n):
        j = 1
        while(j <= n):
            print(count, end='')
            j = j+1
            count = count + 1
        print(end='\n')
        i = i + 1
    

PatternFive(n)


